gtest/src/manager.rs

```diff
@@ -402,13 +393,58 @@ impl ExtManager {
 
     /// Call non-void meta function from actor stored in manager.
     /// Warning! This is a static call that doesn't change actors pages data.
-    pub(crate) fn call_meta(
+    pub(crate) fn read_state(&mut self, program_id: &ProgramId) -> Result<Vec<u8>> {
+        let (actor, _balance) = self
+            .actors
+            .get_mut(program_id)
+            .ok_or_else(|| TestError::ActorNotFound(*program_id))?;
+
+        let (_, program, memory_pages) = actor
+            .get_executable_actor_data()
+            .ok_or_else(|| TestError::ActorIsNotExecutable(*program_id))?;
+
+        core_processor::informational::execute_for_reply::<Ext, WasmiEnvironment<Ext, _>, _>(
+            String::from("state"),
+            program.code().clone(),
+            Some(memory_pages),
+            Some(program.allocations().clone()),
+            Some(*program_id),
+            Default::default(),
+            u64::MAX,
+            self.block_info,
+        )
+        .map_err(TestError::ReadStateError)
+    }
+
+    pub(crate) fn read_state_with_map(
```

Comment:
lets go with similar to pallet and rpc calls naming pls + wrappers for decoded variants
"mapping_" wasm/arg isn't intuitive enough for me